src/styles/base.css

```diff
@@ -55,11 +55,15 @@
   max-width: 88%;
 }
 
-/* Add some better default styling to videos */
-.sl-markdown-content video {
+/* Add some better default styling to videos, images */
+.sl-markdown-content video,
+.sl-markdown-content img {
+  width: 100%;
   max-width: 720px;
-  margin: theme(margin.8);
+  border: 0;
+  margin: 16px 0 32px 0;
   aspect-ratio: 16 / 9;
+  object-fit: cover;
```

Comment:
Two reasons I restyled images and videos:
1. They are not responsive currently, overflowing and messing up layout on mobile
2. Removing margin aligns them with code snippets and notes, which I thought looked a bit nicer (plus bigger videos is helpful given the UI is pretty small)

Reply:
A future improvement might be to set the background color of the video containers to the same background color as the vscode theme they were recorded with. Just would make it less jarring when they popped in. This isn't necessary to do in this PR.

Reply:
You mean set the theme of VS Code in the recording itself to match the theme chosen by the user in Astro docs? I agree and have thought the same thing about code examples on my personal site. Went with dark mode because a splash of dark for a light mode user is way less jarring than the opposite.

Reply:
The Astro theme can switch between light and dark mode, but the video won't. We should just have a consistent theme we record videos on and then set the background of the video container element to have the same background as the video. It's really just to avoid an empty area that pops into the video color.

Reply:
Sorry, but still not sure I follow. Do you mean the thin dark gray border (my Arc browser theme color)? Or do you mean add padding back on ScreenStudio and choose a default background for that?

<img width="762" alt="Screenshot 2024-06-11 at 9 55 22 AM" src="https://github.com/membrane-io/docs/assets/73900714/0476170b-ab23-464f-907e-9c3e428457de">